### DISCUSSION TITLE
Convert Kubernetes YAML into `.pod` and `.container` Quadlet files

### DISCUSSION THREAD

**Discussion by byrnedo** [k9withabone/podlet#113] [Ideas]
I would find it useful to be able to convert a k8s yaml file containing a pod into a .pod file.

**Comment by k9withabone** [comment]
Interesting idea! Theoretically, it wouldn't be technically difficult, but rather tedious work. For me this lands in the "someday maybe?" category in terms of priority, which is why I moved this to a discussion. As per usual I would gladly accept a PR if someone else wants to put in the work before I eventually get to it. Before someone starts working on it though, I would like to talk to them about their implementation plan.

**Reply by byrnedo** [reply]
Yeah I understand the tediousness of it. Really what I want it an escape hatch between `podman kube play` and the actual pod creation. As in if it could convert to the equivalent .pod or command line. As of now there are aspects of running with gpus that don't work yet, but that I could manually fixup.
